packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1930,6 +1930,18 @@ export class MergeTree {
         }
     }
 
+    public obliterateRange(
+        start: number,
+        end: number,
+        refSeq: number,
+        clientId: number,
+        seq: number,
+        overwrite = false,
+        opArgs: IMergeTreeDeltaOpArgs,
+    ): void {
+        this.markRangeRemoved(start, end, refSeq, clientId, seq, overwrite, opArgs);
```

Comment:
since it will be a while before we implement this, I'd prefer we inject this behavior as part of the test suite. If we really want to add it here, we should document that it's not correctly implemented yet and to see the design doc.